doc/manual.md

```diff
@@ -6549,16 +6549,16 @@ The `include` statement can be used outside the top level, as such:
 Module names in imports
 -----------------------
 
-A module alias can be introduced via the `as` keyword:
+A module alias can be introduced via the `as` keyword (the original module name
+is not accessible then):
```

Comment:
I'm particularly in favor of this change (and the one below). To me it feels grammatically awkward, as the parenthesized statement ("the original module name is not accessible then") interrupts the connection between the original statement ("A module alias...") and the example.

Reply:
Change to:

> A module alias can be introduced via the `as` keyword, and the original module name
> is not accessible then: